CHANGELOG.md

```diff
@@ -1,4 +1,11 @@
 ## 0.18.0 (Unreleased)
+
+## 0.17.1 (June 24, 2020)
```

Comment:
```suggestion
## 0.17.1 (Unreleased)
```

Reply:
Ah okay so that has to be Unreleased. Until the release is actually cut.